site/src/docs/components/cookie-consent/api.mdx

```diff
@@ -0,0 +1,135 @@
+| `onAllConsentsGiven`                                                                                                             | No       | See Callbacks section below.                                                                                                                                                                                                                                                                                           |
+| `onConsentsParsed`                                                                                                               | No       | See Callbacks section below.                                                                                                                                                                                                                                                                                           |
+| `noCommonConsentCookie`                                                                                                          | No       | Consents are saved to a cookie and this option can prevent showing that cookie in the list of consents. If this is not true, `requiredConsents` are always shown and a common group “sharedConsents” is added to it and also a common cookie “helConsentCookie” to that group. All content for those is in `content.json`. |
+| `focusTargetSelector`                                                                                                            | No       | When the modal is closed or the ESC key is pressed when it is open, the focus must be shifted outside the modal. This defines the element to set focus to with this selector (string). Not needed when modal is not used. Throws an error is not set for modal.                                                                |
+[Table 1:ContentSource object properties]|
+
+##### ContentSource examples
+You can find examples of ContentSource property contents in <Link href="https://github.com/NikoHelle/helsinki-design-system/blob/feature/cookie-consent/packages/react/src/components/cookieConsent/CookieConsent.stories.tsx" external>the HDS CookieConsent Storybook examples (FIX LINK)</Link>.
+
+#### CookieDomain 
+Rarely used. Cookie consents are stored to a cookie whose domain is automatically set to accept subdomains: *.hel.fi or *.hel.ninja or *.domain.com. Some sites are included in the public suffix list and cookies for those sites cannot accept subdomain cookies. Github.io is one of those sites and HDS examples do not work there unless the cookie domain can be overridden. Therefore `cookieDomain` was added.
+
+#### Detailed info about properties
+
+##### Required and optional cookie categories
```

Comment:
Reading this it's little unclear that which properties these are? The main api seems to have only two properties: `contentSource` and `cookieDomain`. Are these part of the `contentSource` property? Maybe we could change the header to something like:

#### contentSource.requiredConsents / contentSource.optionalConsents

Reply:
Fixed